diff --git a/dist/createJestConfig.js b/dist/createJestConfig.js
index ec9529873282ea4cac471efb79c9788747b62317..03bf4bfc7fbcc5553f007fc01d45d93548eb3d44 100644
--- a/dist/createJestConfig.js
+++ b/dist/createJestConfig.js
@@ -10,7 +10,7 @@ function createJestConfig(_, rootDir) {
         moduleFileExtensions: ['ts', 'tsx', 'js', 'jsx', 'json', 'node'],
         collectCoverageFrom: ['src/**/*.{ts,tsx,js,jsx}'],
         testMatch: ['<rootDir>/**/*.(spec|test).{ts,tsx,js,jsx}'],
-        testURL: 'http://localhost',
+        testEnvironmentOptions: { url: 'http://localhost' },
         rootDir,
         watchPlugins: [
             require.resolve('jest-watch-typeahead/filename'),
diff --git a/templates/react/example/.gitignore b/templates/react/example/.gitignore
deleted file mode 100644
index 587e4ec7aec54ca7894daa54538d67aefe8541f7..0000000000000000000000000000000000000000
diff --git a/templates/react-with-storybook/example/.gitignore b/templates/react-with-storybook/example/.gitignore
deleted file mode 100644
index 587e4ec7aec54ca7894daa54538d67aefe8541f7..0000000000000000000000000000000000000000
